<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-U74Q3VS2" name="TC002(failed)" tests="1" failures="0" timestamp="20 Jun 2020 07:37:56 GMT" time="0.005" errors="0">
  <testcase name="@BeforeClass beforeClassTestAnnLoad" time="0.0" classname="in.ibm.chirra.saf.test.Test_2">
    <failure type="java.io.FileNotFoundException" message="\TC002_2020_06_20_13_07_56.log (Access is denied)">
      <![CDATA[java.io.FileNotFoundException: \TC002_2020_06_20_13_07_56.log (Access is denied)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.apache.log4j.FileAppender.setFile(FileAppender.java:294)
	at org.apache.log4j.FileAppender.<init>(FileAppender.java:110)
	at org.apache.log4j.FileAppender.<init>(FileAppender.java:121)
	at in.ibm.chirra.saf.driverSetup.DriverSetup.beforeClassTestAnnLoad(DriverSetup.java:71)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeClass beforeClassTestAnnLoad -->
  <testcase name="@AfterClass afterClassTestAnnLoad" time="0.0" classname="in.ibm.chirra.saf.test.Test_2">
    <skipped/>
  </testcase> <!-- @AfterClass afterClassTestAnnLoad -->
  <testcase name="sampleTest1" time="0.0" classname="in.ibm.chirra.saf.test.Test_2">
    <skipped/>
  </testcase> <!-- sampleTest1 -->
</testsuite> <!-- TC002(failed) -->
